src/common/com/intellij/plugins/haxe/ide/refactoring/introduce/HaxeIntroduceHandler.java

```diff
@@ -412,15 +474,20 @@ protected PsiElement createDeclaration(Project project, String text, PsiElement
   private PsiElement performReplace(@NotNull final PsiElement declaration, final HaxeIntroduceOperation operation) {
     final HaxeExpression expression = operation.getInitializer();
     final Project project = operation.getProject();
-    return new WriteCommandAction<PsiElement>(project, expression.getContainingFile()) {
+
+    new WriteCommandAction<PsiElement>(project, expression.getContainingFile()) {
```

Comment:
WriteCommandAction instantiating is deprecated, also as it's execute method. May be it could be replaced with something like:

```java
WriteCommandAction.writeCommandAction(expression.getContainingFile())
   .withName("Replace occurrences with '"+operation.getName() + "'.")
   .run<Exception> {
      ...
   }
```

Reply:
Yes, it is deprecated, as of 2018.3.x.  The Builder API that is suggested wasn't introduced until 2017.1.  Since we're still supporting 2016 versions, and because removing the deprecated function will require divergence, I propose that we put off conversion until we drop 2016, or until the API is removed (whichever comes first).